app/models/invenio_rdm_record_converter.rb

```diff
@@ -511,6 +511,10 @@ def dh_collection_to_prism_community_collection
           # there's a match, no need to keep searching
           if mapping_entry
             return format_prism_community_collection_string(mapping_entry)
+          # part of pediatric neurology brief collection in format pnb-volume#-issue#
+          elsif collection_path_entry[:id]&.include?("pnb")
```

Comment:
What about this little refactoring (it's not exact ruby so correct it)

```ruby
def dh_collection_to_prism_community_collection

  def find_mapping_entry_in_dh_collections
    @dh_collections.map do |collection_path|
      collection_path.each do |collection_path_entry|
        mapping_entry = @@dh_to_prism_entity[collection_path_entry[:id]]

        # there's a match, no need to keep searching
        if mapping_entry
          return mapping_entry
        # part of pediatric neurology brief collection in format pnb-volume#-issue#
        elsif collection_path_entry[:id]&.startswith?("pnb-")  # CHANGE to ruby
          volume_number, issue_number = collection_path_entry[:id].split("-").last(2)
          return {  # Honestly, this 'may' be risky if other valid ones down the road in `@dh_collections` . Perhaps a split phase refactoring is better :shrug:   
            "community_id": "pediatric-neurology-briefs", 
            "collection_id": "Volume #{"%02d" % volume_number}, Issue #{"%02d" % issue_number}"
          }
        end
      end
    end
  end

  mapping_entry = 
    @@dh_to_prism_entity[@generic_file.id] ||
    find_mapping_entry_in_dh_collections ||
    {"community_id": "", "collection_id": ""}

  return format_prism_community_collection_string(mapping_entry)
end
```

and

```ruby
def format_prism_community_collection_string(mapping_entry)
    community_id = mapping_entry["community_id"]
    collection_id = mapping_entry["collection_id"]

    if !community_id.blank? && !collection_id.blank?  #verify ruby syntax
      "#{community_id}::#{collection_id}"
    else
      community_id
    end
  end
```

We should get rid of the `pnb-` entries here https://github.com/galterlibrary/digital-repository/blob/7b0fb2132f0fb0b9103d6fcc200192f9b8ec71e4/dh_collections_prism_collection_community.json#L159 , right?

Reply:
Good suggestion, I'll make these changes